From ba5a16d577d2f176d9e99376f2aa74e26c1239db Mon Sep 17 00:00:00 2001
From: Firoz Khan <firoz.khan@linaro.org>
Date: Tue, 25 Sep 2018 15:58:05 +0530
Subject: [PATCH v2 6/6] ia64: add __IGNORE* entries in asm/unistd.h

Add __IGNORE* entries for perf_event_open, seccomp, pkey_
mprotect, pkey_alloc, pkey_free, statx, io_pgetevents and
rseq in the file asm/unistd.h as it is correct to keep
__IGNORE* entry in non uapi header asm/unistd.h while uapi/
asm/unistd.h must hold information only useful for user
space applications.

Signed-off-by: Firoz Khan <firoz.khan@linaro.org>
---
 arch/ia64/include/asm/unistd.h | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/ia64/include/asm/unistd.h b/arch/ia64/include/asm/unistd.h
index 397b143..a0941e6 100644
--- a/arch/ia64/include/asm/unistd.h
+++ b/arch/ia64/include/asm/unistd.h
@@ -26,6 +26,15 @@
 #define __IGNORE_vfork		/* clone() */
 #define __IGNORE_umount2	/* umount() */
 
+#define __IGNORE_perf_event_open
+#define __IGNORE_seccomp
+#define __IGNORE_pkey_mprotect
+#define __IGNORE_pkey_alloc
+#define __IGNORE_pkey_free
+#define __IGNORE_statx
+#define __IGNORE_io_pgetevents
+#define __IGNORE_rseq
+
 #if !defined(__ASSEMBLY__) && !defined(ASSEMBLER)
 
 #include <linux/types.h>
-- 
1.9.1

